Automattic/jetpack
# Add Jetpack icon to gutenberg features powered by Jetpack

This change helps show the user which Gutenberg blocks are powered by Jetpack by putting a little Jetpack icon in the corner of the block's editor icon.

#### Changes proposed in this Pull Request:

* Add Jetpack icon in corner of blocks powered by Jetpack.

#### Testing instructions:

* Install Gutenberg
* Edit a post
* Open the "insert block" dialog
* Search for "VR"
* Should have a little green icon indicating it's a Jetpack feature :)

#### Screenshot

<img width="507" alt="jetpack-gutenberg-icon" src="https://user-images.githubusercontent.com/51896/40245919-d46c93de-5a7b-11e8-8b61-3fec66262d98.png">
